### DISCUSSION TITLE
✅ Elementor is lacking a flexible content carousel

### DISCUSSION THREAD

**Discussion by TrojanProxy** [elementor/elementor#15366] [Feature Request]
Elementor is lacking a flexible content carousel. We have an image carousel, but it would be really useful to have a carousel that can hold some text and a button.

**Comment by nicholaszein** [comment]
Hello there !

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.14! 🥳 

You can now use the new [Nested Carousel](https://elementor.com/help/carousel-widget/) to create your own layouts.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
